lib/Dialect/FIRRTL/Transforms/EmitOMIR.cpp

```diff
@@ -327,42 +327,6 @@ void EmitOMIRPass::runOnOperation() {
   if (anyFailures)
     return signalPassFailure();
 
-  // Delete the temporary NLAs. This requires us to visit all the nodes along
-  // the NLA's path and remove `circt.nonlocal` annotations referring to the
-  // NLA.
-  DenseSet<InstanceOp> instsToModify;
-  DenseSet<StringAttr> nlaSymNamesToDrop;
-
-  for (auto nla : removeTempNLAs) {
-    LLVM_DEBUG(llvm::dbgs() << "Removing '" << nla << "'\n");
-    nlaSymNamesToDrop.insert(nla.sym_nameAttr());
-    for (auto nameRef : nla.namepath()) {
-      if (auto innerRef = nameRef.dyn_cast<hw::InnerRefAttr>()) {
-        auto it = instancesByName.find(innerRef);
-        if (it == instancesByName.end())
-          continue;
-        instsToModify.insert(it->second);
-      }
-    }
-    nlaTable->erase(nla);
-    nla.erase();
-  }
-
-  for (auto instOp : instsToModify) {
-    AnnotationSet::removeAnnotations(instOp, [&](Annotation anno) {
-      auto match =
-          anno.isClass("circt.nonlocal") &&
-          nlaSymNamesToDrop.contains(
-              anno.getMember<FlatSymbolRefAttr>("circt.nonlocal").getAttr());
-      if (match)
-        LLVM_DEBUG(llvm::dbgs()
-                   << "Removing " << anno.getDict() << " from "
-                   << instOp->getParentOfType<FModuleOp>().getName() << "."
-                   << instOp.name() << "\n");
-      return match;
-    });
-  }
-
```

Comment:
Thanks for removing this redundant code, should have been removed after the NLATable mover over.